models/articles.models.js

```diff
@@ -1,18 +1,31 @@
-const db = require('../db/connection');
+const db = require("../db/connection");
 
 function fetchArticleById(article_id) {
-  return db.query(
-  `
+  return db
+    .query(
+      `
     SELECT * FROM articles WHERE article_id = $1
   `,
-  [article_id]
-  )
-  .then(res => {
-    if (res.rowCount === 0) {
-      return Promise.reject({ status: 404, msg: 'article not found'});
-    }
-    return res.rows[0];
-  });
+      [article_id]
+    )
+    .then((res) => {
+      if (res.rowCount === 0) {
+        return Promise.reject({ status: 404, msg: "article not found" });
+      }
+      return res.rows[0];
+    });
 }
 
-module.exports = { fetchArticleById };
\ No newline at end of file
+function fetchArticles() {
+  return db
+    .query(`
+      SELECT a.*, count(c.comment_id)::int AS comment_count
```

Comment:
could you tweak this so that you don't return the body - just to match the ticket